### DISCUSSION TITLE
Testing my FundMe Contract on another Wallet

### DISCUSSION THREAD

**Discussion by yeahChibyke** [Cyfrin/foundry-full-course-cu#407] [Q&A]
I built a close clone of the FundMe smart contract from Lesson 4. I connected my Trust Wallet, but finding the Sepolia Testnet on there is a very big headache. I finally discovered Chainlist, and connected my Trust Wallet to add the network. I don't know how safe Chainlist is. And it seems my suspicions are true; I sent 1 Sepolia ETH to the Sepolia address on Trust Wallet, the transaction is successful as of nearly 10-15 mins ago, but it is yet to reflect on Trust Wallet. Should I wait longer?
Plus, is there a safer and verified way to add Sepolia Testnet to Trust Wallet?

**Comment by Naman1729** [comment]
Hey @yeahChibyke , One of the trusted methods to add a network in MetaMask is through Alchemy. However, before proceeding, it is essential to ensure that the Sepolia testnet is not already present by default in MetaMask.
Go to this link :
https://www.alchemy.com/overviews/how-to-add-sepolia-to-metamask

**Reply by yeahChibyke** [reply]
I already have Sepolia available on Metamask, and I used it to successfully deploy and test the smart contract. I wanted to use another wallet to confirm the receive and fallback functions, that's why I went with Chainlink. I sent some Sepolia ETH from my Metamask to my Trust wallet, and despite the fact that the transaction is successful according to Etherscan, it is yet to reflect on Trust. I feel scammed 😐
